src/runtime-rs/crates/resource/src/volume/mod.rs

```diff
@@ -11,23 +11,22 @@ mod share_fs_volume;
 mod shm_volume;
 pub mod utils;
 
-pub mod vfio_volume;
-use vfio_volume::is_vfio_volume;
-
+mod direct_volume;
 pub mod spdk_volume;
-use spdk_volume::is_spdk_volume;
+pub mod vfio_volume;
```

Comment:
From the current design, the direct volume includes vfio volume, spdk volume, spool volume, I think it's better to add the sub-volume type into a directory
```
|-- directory_volume
  |-- vfio_volume.rs
  |-- spdk_volume.rs
  |-- spool_volume.rs
```

Reply:
fixed